openmp/libomptarget/plugins-nextgen/common/src/PluginInterface.cpp

```diff
@@ -790,7 +790,7 @@ Error GenericDeviceTy::deinit(GenericPluginTy &Plugin) {
                              &ImageDeviceMemoryPoolTracking);
       if (auto Err =
               GHandler.readGlobalFromDevice(*this, *Image, TrackerGlobal))
-        return Err;
+        continue;
```

Comment:
Needs to be consumed here. In the future we should make a lookup function that doesn't Error.

Reply:
Isn't checking it enough? Why didn't I see an error... hmm

Reply:
Checking is enough if it's `success()` not if it's an Error.